Dockerfile

```diff
@@ -19,6 +19,11 @@ COPY secrets /opt/ml-api-adapter/secrets
 
 FROM node:12.16.0-alpine
 
+WORKDIR /opt/central-ledger/cert
+RUN apk add --no-cache curl 
+RUN curl --etag-compare etag.txt --etag-save etag.txt --remote-name https://curl.se/ca/cacert.pem \
```

Comment:
Do we really need to add this as part of the Docker build?

I feel like this should rather be mounted by the Docker/Kubernetes runtime.

Reply:
Fair enough, we can arrange for that to happen. Then these parts not needed.